doc/users/next_whats_new/Allow positioning of text inside TextBox.rst

```diff
@@ -0,0 +1,16 @@
+Text can be positioned inside TextBox widget
+---------------------
+
+A new parameter called ``textalignment`` can be used to control for the position of the text inside the Axes of the TextBox widget.
+
+Example :
+
+.. code-block::
```

Comment:
This can be made visible in the entry:
```suggestion
.. plot::
```
Then you may want to show all 3 alignments.